tests/modules/cnvkit/batch/test.yml

```diff
@@ -0,0 +1,101 @@
+- name: cnvkit batch test_cnvkit_tumoronly
+  command: nextflow run tests/modules/cnvkit/batch -entry test_cnvkit_tumoronly -c tests/config/nextflow.config
+  tags:
+    - cnvkit
+    - cnvkit/batch
+  files:
+    - path: output/cnvkit/reference.antitarget-tmp.bed
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
+    - path: output/cnvkit/reference.target-tmp.bed
+      md5sum: 26d25ff2d6c45b6d92169b3559c6acdb
+
+- name: cnvkit batch test_cnvkit_hybrid
+  command: nextflow run tests/modules/cnvkit/batch -entry test_cnvkit_hybrid -c tests/config/nextflow.config
+  tags:
+    - cnvkit
+    - cnvkit/batch
+  files:
+    - path: output/cnvkit/baits.antitarget.bed
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
+    - path: output/cnvkit/baits.target.bed
+      md5sum: 26d25ff2d6c45b6d92169b3559c6acdb
+    - path: output/cnvkit/reference.cnn
+      md5sum: ac99c1ad8b917b96ae15119146c91ab9
+    - path: output/cnvkit/test.paired_end.sorted.antitargetcoverage.cnn
+      md5sum: 203caf8cef6935bb50b4138097955cb8
+    - path: output/cnvkit/test.paired_end.sorted.bintest.cns
+      md5sum: 6544d979475def8a9f69ba42a985668d
+    - path: output/cnvkit/test.paired_end.sorted.call.cns
+      md5sum: f2ca59b4d50b0c317adc526c1b99b622
+    - path: output/cnvkit/test.paired_end.sorted.cnr
+      md5sum: 7e37d73ab604dbc3fe4ebb56aca9bdc3
+    - path: output/cnvkit/test.paired_end.sorted.cns
+      md5sum: 060af1aa637ed51812af19bcce24fcfe
+    - path: output/cnvkit/test.paired_end.sorted.targetcoverage.cnn
+      md5sum: 3fe80b6013ffc3e9968345e810158215
+    - path: output/cnvkit/test.single_end.sorted.antitargetcoverage.cnn
+      md5sum: 203caf8cef6935bb50b4138097955cb8
+    - path: output/cnvkit/test.single_end.sorted.targetcoverage.cnn
+      md5sum: aa8a018b1d4d1e688c9f9f6ae01bf4d7
+
+- name: cnvkit batch test_cnvkit_wgs
+  command: nextflow run tests/modules/cnvkit/batch -entry test_cnvkit_wgs -c tests/config/nextflow.config
+  tags:
+    - cnvkit
+    - cnvkit/batch
+  files:
+    - path: output/cnvkit/genome.antitarget.bed
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
```

Comment:
also empty target bed. If it is expected then please ignore my statements. in that case maybe a comment line would help, although it would not get generated with the create-test-yml

Reply:
That's fine - it's good to memorize that `d41d8cd98f00b204e9800998ecf8427e`md5sum I guess. It seems with the small datasets we are running in test it's common that the antitarget.bed will be empty. I tried to re-run with a bigger `tumorbam` and then it's not empty anymore. I also tried to run with `'args': '--output-reference reference.cnn --antitargets /home/lasse/test.bed'` i.e. a local antitargets.bed arguments, (just used the one from [sars2](https://github.com/nf-core/test-datasets/blob/de06de5a9b4b778cff579437f516f4f64b66a789/data/genomics/sarscov2/genome/bed/test.bed)). Then the empty file problem disappears, because it doesn't output antitargets anymore. As expected.  So I don't see a good way around it, other than large input test data. Which is not good either.